config/v2/config.go

```diff
@@ -428,6 +443,20 @@ func (c *Config) Generate(r *rand.Rand, size int) reflect.Value {
 		return nil
 	}
 
+	randKubernetesProvider := func(r *rand.Rand, s int) *KubernetesProvider {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`(*Config).Generate$13` - `s` is unused (unparam)